scripts/_common.sh

```diff
@@ -5,8 +5,8 @@
 #=================================================
 
 # dependencies used by the app
-pkg_dependencies="postgresql python3"
-extra_dependencies="libunixsocket-java signald>=0.15.0-27 signaldctl"
+pkg_dependencies="postgresql python3 python3-dev build-essential"
+extra_dependencies="libunixsocket-java signald signaldctl"
```

Comment:
I would like to pin a signald version to avoid breaking updates, as well as allow installing a previous version of this repository, including for CI operations.

Reply:
I'd like too, but I'm not sure the helper now supports this. It is already great if it upgrades at all.You can test and add this in a next  PR.